research/train.py

```diff
@@ -0,0 +1,42 @@
+import argparse
```

Comment:
Added python3 shebang since we have no intention of supporting python 2. Similarly, I didn't add the coding header since we only care about Python 3 and it uses unicode by default.